### DISCUSSION TITLE
What licenses would you recommend for openapi schema?

### DISCUSSION THREAD

**Discussion by rafalkrupinski** [OAI/OpenAPI-Specification#3050] [Q&A]
What license would you recommend for API schema that would free the client code generated from any legal burden?

Some might argue that a generated client is a translation or other derivative work. distributing the schema under CC SA for example, might impose the license on the generated client.

Any recommendations?

**Comment by handrews** [comment]
We are not lawyers and cannot answer legal questions (about licensing or anything else).
